packages/utils/tests/validators.test.ts

```diff
@@ -0,0 +1,31 @@
+import { isASCII, hasUppercase } from '../src/validators';
+
+describe('utils/suite/validators', () => {
+    describe('isASCII', () => {
+        it('should return true for ASCII only string', () => {
+            expect(isASCII('this is only ascii')).toEqual(true);
+        });
+
+        it('should return true when called without parameter', () => {
+            expect(isASCII()).toEqual(true);
+        });
+
+        it('should return false strings with non ASCII chars', () => {
+            const fooStrings = ['¥', 'železniční přejezd'];
```

Comment:
nit: since there just two string, then maybe will be simplier to just explicitly define them `expect(isASCII('železniční přejezd')).toEqual(false)` and I believe it will also throw little bit better error message in case of fail